challenges/2021-05-18-sum-of-squares/solutions/go/drkennetz/solution_test.go

```diff
@@ -0,0 +1,39 @@
+package solution
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestSumOfSquares(t *testing.T) {
+	tables := []struct {
+		input int
+		result int
+	}{
+		{13, 2},
+		{27, 3},
+		{144, 1},
+		{84, 3},
+		{85, 2},
+	}
+	for _, x := range tables {
+		soln := SumOfSquares(x.input)
+		if soln != x.result {
+			t.Error("FAIL: SumOfSquares of: ", x.input, "was incorrect, got: ", soln, "expected: ", x.result)
+		} else {
+			fmt.Println("SUCCESS: SumOfSquares of: ", x.input, "yielded: ", soln, "which was correct!")
+		}
+	}
+}
+
+func benchmarkSumOfSquares(i int, b *testing.B) {
```

Comment:
Nice addition for benchmark to show the stats.

Reply:
Thanks! I just learned about how to run benchmarks on multiple inputs so I thought I'd give it a go.